addon/index.js

```diff
@@ -0,0 +1,40 @@
+/* globals require, exports */
+"use strict";
+const self = require("sdk/self");
+const shield = require("./shield-utils/index");
+const {when: unload} = require("sdk/system/unload");
+const {Feature} = require("./shield");
+const feature = new Feature();
+
+const studyConfig = {
+  name: "ACTIVITY_STREAM",
+  days: 14,
+  variations: {
+    "ActivityStream": () => feature.loadActivityStream(),
+    "Tiles": () => feature.loadTiles()
+  }
+};
+
+class OurStudy extends shield.Study {
+  constructor(config) { // eslint-disable-line no-useless-constructor
+    super(config);
```

Comment:
Any reason not to just remove the constructor and have it automatically delegate?